src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -1113,8 +1113,19 @@ public void updateInstallation(int id, String json) {
                 logger.debug("Request {} {}", request.getMethod(), request.getUrl());
             }
             return client.executeRequest(request, handler);
-        } catch (IOException e) {
-            throw new ZendeskException(e.getMessage(), e);
+        } catch (Throwable t) {
+            throw new ZendeskException(t);
+        }
+    }
+
+    private static abstract class ZendeskAsyncCompletionHandler<T> extends AsyncCompletionHandler<T> {
+        @Override
+        public void onThrowable(Throwable t) {
+            if (t instanceof IOException) {
```

Comment:
might be worth logging something here or throwing ZendeskException for all exceptions, super just chews it / logs debug.